ansible/vars/topo_t0.yml

```diff
@@ -64,13 +64,13 @@ topology:
         Vlan100:
           id: 100
           intfs: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]
-          prefix: 192.168.100.1/21
+          prefix: 192.168.0.1/22
```

Comment:
>prefix: 192.168.0.1/22 [](http://example.com/codeflow?start=10&length=22)

The change looks good to me. I believe there are several rules to pick a correct prefix. Is it possible to add the rules into a checker?

Some from thought:
1. prefix should be valid and strict
2. prefix length should be large enough (what is the min?)
3. prefix range should not conflict
4. check both v4 and v6.

Some possible places to add checkers:
1. gen_mg
2. minigraph_facts
3. PR checker will be even better for this kind of config mistakes.

Reply:
I agree that a checker would be more efficient for catching config issues. I think that could probably be a future task and needs a bit of thinking.

Reply:
Create an issue?

Reply:
Sure. Issue created here https://github.com/Azure/sonic-mgmt/issues/3912